suite-native/module-home/src/components/PortfolioGraphHeader.tsx

```diff
@@ -37,81 +44,103 @@ export const writeOnlyReferencePointAtom = atom<null, ExtendedGraphPoint>(
     },
 );
 
-const percentageDiff = (a: number, b: number) => {
-    if (a === 0 || b === 0) return 0;
-    return 100 * ((b - a) / ((b + a) / 2));
-};
-
-const percentageChangeAtom = atom(get => {
+export const percentageChangeAtom = atom(get => {
     const selectedPoint = get(selectedPointAtom);
     const referencePoint = get(referencePointAtom);
     return percentageDiff(referencePoint.value, selectedPoint.value);
 });
 
-const Balance = () => {
-    const { FiatAmountFormatter } = useFormatters();
-    const [point] = useAtom(selectedPointAtom);
-
-    return <FiatAmountFormatter value={point.value} />;
-};
-
-const getColorForPercentageChange = (percentageChange: number) =>
```

Comment:
Can put this back? This code is repeated here

Reply:
repetition removed